### DISCUSSION TITLE
Discussion Feedback

### DISCUSSION THREAD

**Discussion by thanhcsf** [d2l-ai/d2l-vi#2549] [General]
Mình tạo theard đầu tiên để thử nghiệm và ghi nhận các ý tưởng phát triển đối với Github Discussions. Anh em có thể thử nghiệm tính năng và đóng góp ý tưởng tại đây nhé.

**Comment by lkhphuc** [comment]
Awesome babe. Chắc glossary mình move sang đây thảo luận hết nhỉ, rồi link discussion tới một trong các thread này.

**Reply by lkhphuc** [reply]
Threaded discussion chắc dễ follow hơn.
Với lại chắc hoạt động trên slack cũng sẽ move qua bên này, nhiều người theo dõi được hơn.

**Reply by thanhcsf** [reply]
Yeah, ý mình cũng là thế. Mình cũng không yêu cầu tham gia vào Slack được nhiều, mà vào rồi cũng có ai nói năng gì đâu, ở đây là ổn rồi nhỉ :))

**Reply by minhduc0711** [reply]
cái discussion này chuẩn rồi, tại em cũng lười check slack :smile:

**Reply by rootonchair** [reply]
Hic, mình là team siêng check slack 😢

**Comment by thanhcsf** [comment]
@trGiang99 Những topic về thuật ngữ trong hình dung của anh nó sẽ luôn mở như vậy và không phải một vấn đề đã được giải quyết. Vì có thể (biết đâu đấy) chúng ta sẽ quay lại thảo luận một lần nữa. Trong Github Discussions khi mark một câu trả lời thì nó cũng giống như PR đã được merged vậy. 

Nên anh thống nhất như vậy cho tách bạch khỏi những topic đặt câu hỏi và dự án trả lời nghen.

Trong thời gian tới thì phần hạng mục cũng sẽ được customize lại cho phù hợp, anh sẽ tách riêng một hạng mục cho thảo luận thuật ngữ, và mình cứ thảo luận thôi, không cần mark as answer.

**Reply by trgiangdo** [reply]
Ok anh. Em thấy vậy cũng hợp lý.

**Comment by minhduc0711** [comment]
@duythanhvn không biết có cách nào để gửi notification tự động (giống kiểu tự động request review khi mở PR) không anh nhỉ?

**Reply by thanhcsf** [reply]
Ừa, cái này anh cũng đang thảo luận thôi. Vì nói cho ngay cũng như một mini-forum mà có một block riêng để assigner thì nghe màu cũng kỳ. Phương án khả dĩ hiện tại anh thấy là trong bản cập nhật tiếp theo (có thể là tháng tới) thì anh cập nhật được danh sách các hạng mục và template cho từng cái đấy, anh sẽ để sẵn mentions đến các bạn theo từng hạng mục.

Ví dụ như thảo luận thuật ngữ thì mention em và các anh em đã có trong danh sách CODEOWNERS.

Tạm thời anh sẽ thủ công việc mention cho anh em vào các topic mới nhé.
